src/main/webapp/scripts/module/purap/reqsNotes.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Forcibly sets the "Copy Note to PO?" checkbox on the REQS note,
+ * and also hides the checkbox and shows the text "Yes". Intended to be called
+ * after an attachment file has been specified on the note addLine.
+ */
+function forceNewNoteCopyToPO() {
+    var copyToPOElem = document.getElementById("newNote.extension.copyNoteIndicator");
+    var forceCopyYesLabel = document.getElementById("newNoteForceCopyYesLabel");
+    if (copyToPOElem) {
+        copyToPOElem.checked = "checked";
+        copyToPOElem.style.display = "none";
+        if (forceCopyYesLabel) {
+            forceCopyYesLabel.style.display = "inline";
+        }
+    }
+}
```

Comment:
Not that I'm aware of, unless we want to forcibly overwrite the copy-to-PO flag on the server after the form gets submitted. As per the user story, the functionals want notes with attachments to copy to the PO, so this seemed like the most user-friendly way of doing that.

I may need to verify with the functionals whether attachments created after REQS finalization should still copy to the PO if the document did not become an APO.